pkg/routes/routes_test.go

```diff
@@ -19,14 +21,19 @@ import (
 	"github.com/storj/dms/pkg/storage"
 )
 
+var (
+	c         *chance.Chance
+	usedPorts []int
```

Comment:
This will fail with concurrent executions of tests.

If you control the listener, then using `127.0.0.1:0` will work better in most cases.